jcl/src/java.base/share/classes/java/lang/StringBuilder.java

```diff
@@ -2261,7 +2261,47 @@ public StringBuilder append(CharSequence sequence, int start, int end) {
 	if (sequence == null) {
 		return append(String.valueOf(sequence), start, end);
 	} else if (sequence instanceof String) {
+		/*[IF Sidecar19-SE]*/
```

Comment:
I don't think we need a Java 9 path here any longer. This file will never be compiled for Java 9+ anyway.

Reply:
Thanks, removed the Java 9 path.